### ISSUE TITLE
Facebook social login not returning email

### ISSUE THREAD

**Issue by Ashar2shahid** [Spyna/protect-rest-api-after-social-login#1]
I was going through your excellently written medium article, and while following along ran into this issue your api call : 
`https://graph.facebook.com/v2.11/' + user_id + '?fields=id,name,picture,email&access_token=' + appToken`     
This call was not returning the email, i tried updating to v3.2 of the api as well as adding &scope=email in the params and it still didn't work,

finally i did this "https://graph.facebook.com/" +"me" + "?fields=id,name,email,picture&access_token=" +user_access_token + "&scope=email" and it worked (user_access_token being code in your case). Is this alright? is there any security issue with this?

**Comment by Spyna** [comment]
Yes, it should be fine. Not sure if `/me` only works for "me" or also for the other users.
